apps/docs/content/components/status/FloatingBadge.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: FloatingBadge
+description: A floating badge allows the user to position a badge relative to another component.
+category: "status"
+links:
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/badge/src/FloatingBadge.tsx
+---
+
+<Example src="Badge/docs/floatingbadge/preview" isOpen />
+
+## Props
+
+<PropTable component="FloatingBadge" />
+
+<FeatureFlag flag="next">
+    ## Guidelines
+
+    TODO: If we have some guidelines about this component&apos;s usage
+
+    ### Accessibility?
+
+    TODO: If we have some guidelines about this component and accessibility
+</FeatureFlag>
+
+<FeatureFlag flag="rc">
+    ## Anatomy
+
+    TODO: We have anatomy screenshots from the Figma, we could most likely use them here
+
+    ### Concepts
+
+    TODO: links to related concepts
+</FeatureFlag>
+
+## Examples
+
+### Placement
+
+A floating badge can be positioned relative to another component using the `placement` prop. This allows the badge to be placed in any of the 4 positions around the target component.
+
+<Example src="Badge/docs/floatingbadge/placement" />
+
+### Overlap
+
+Allows to specify if the badge is overlaping a rectangular or circular component.
```

Comment:
While I may have written something like this before, it sounds weird to me. Maybe we could say either:
"Allows specifying whether the badge overlaps a rectangular or circular component"
or
"Allows you to specify whether the badge overlaps a rectangular or circular component"